packages/web-components/src/components/masthead/defs.ts

```diff
@@ -54,3 +54,19 @@ export enum MEGAMENU_LAYOUT_SCHEME {
   TAB = 'tab',
   LIST = 'list',
 }
+
+/**
+ * Matches CMaaS's CTA options.
+ *
+ * @see https://github.ibm.com/live-advisor/cm-app/blob/master/docs/cm-doc.md#calls-to-action
+ */
+export enum L1_CTA_TYPES {
```

Comment:
...weren't we trying to avoid ENUMs for some reason?

Reply:
Yes, but they're idiomatic to this library. Figured I'd go with them instead of making a special snowflake.